translations/fr.json

```diff
@@ -699,22 +671,21 @@
   "gamestatus.destroyed": "détruite(s)",
   "gamestatus.tasks": "Tâches {0}/{1}",
   "gamestatus.tasks.complete": "Complétée!",
-
   "settings.title": "PARAMÈTRES",
   "settings.volume": "Volume sonore {0}",
   "settings.soundeffects": "Effets sonores",
   "settings.ambience": "Ambiance",
-  "settings.tribemusic": "Son de tribu",
-  "settings.suggestions": "Suggestions de jeu",
+  "settings.tribemusic": "Musique de tribu",
```

Comment:
The same problem with "arbre de recherche", it causes dysfunctions between buttons.